src/import/sst_service.rs

```diff
@@ -164,6 +164,8 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
         let timer = Instant::now_coarse();
         let importer = Arc::clone(&self.importer);
         let limiter = self.limiter.clone();
+        let engine = Arc::clone(&self.engine);
+        let sst_writer =<RocksEngine as SstExt>::SstWriterBuilder::new().set_db(RocksEngine::from_ref(&engine)).build(self.importer.get_path(req.get_sst()).to_str().unwrap()).unwrap();
```

Comment:
```suggestion
        let sst_writer = <RocksEngine as SstExt>::SstWriterBuilder::new().set_db(RocksEngine::from_ref(&engine)).build(self.importer.get_path(req.get_sst()).to_str().unwrap()).unwrap();
```

Consider applying `rustfmt` on this statement